package.json

```diff
@@ -127,7 +127,7 @@
     "l10n:check": "npm run l10n:extract && git diff-files --quiet --ignore-all-space locales && echo 'No missing strings'",
     "test:ci": "cross-env NODE_ENV=test nyc mocha --recursive --reporter mocha-junit-reporter --require frontend/test-setup.js frontend/test",
     "test:watch": "npm test -- --watch",
-    "test:all": "npm run lint && npm run flow && npm run test && npm run flow-coverage && cd addon && npm run lint && npm run coverage && npm run test && tox -e ui-tests",
+    "test:all": "npm run lint && npm run flow && npm run test && npm run flow-coverage && cd addon && npm run lint && npm run test && tox -e ui-tests",
```

Comment:
This is unrelated, it just fixes test:all now that the addon doesn't have an npm run coverage command any more, but I thought I'd sneak it into this pr.